tools/depends/target/ffmpeg/Makefile

```diff
@@ -26,7 +26,10 @@ ifeq ($(CROSS_COMPILING), yes)
 endif
 ifeq ($(OS), linux)
   ffmpg_config += --target-os=$(OS) --cpu=$(CPU)
-  ffmpg_config += --enable-vdpau --enable-vaapi --enable-pic
+  ifneq ($(TARGET_PLATFORM),raspberry-pi)
+    ffmpg_config += --enable-vdpau
+  endif
+  ffmpg_config += --enable-vaapi --enable-pic
```

Comment:
then --enable-vaapi is definitely going to fail on that special so-called "platform". how didn't it fail before ?